From 91c2ee530598242feb19fde33840233c18e2a508 Mon Sep 17 00:00:00 2001
From: Tobias Mueller <muelli@cryptobitch.de>
Date: Sat, 14 Dec 2019 23:05:39 +0100
Subject: [PATCH 1/2] desktop: Added an initial desktop file

This file will be needed by downstreams.
The path is, obviously, dependent on where the downstram puts it.
One way of editing the file is to execute
desktop-file-edit --set-key Exec --set-value /usr/bin/foo.
---
 src/no.mifi.losslesscut.desktop | 10 ++++++++++
 1 file changed, 10 insertions(+)
 create mode 100644 src/no.mifi.losslesscut.desktop

diff --git a/src/no.mifi.losslesscut.desktop b/src/no.mifi.losslesscut.desktop
new file mode 100644
index 0000000..b714259
--- /dev/null
+++ b/src/no.mifi.losslesscut.desktop
@@ -0,0 +1,10 @@
+[Desktop Entry]
+Name=LosslessCut
+Comment=simple video editor
+Exec=/app/bin/run.sh
+Icon=no.mifi.losslesscut
+Terminal=false
+Type=Application
+Encoding=UTF-8
+Categories=Application;
+StartupWMClass=losslesscut
-- 
2.17.1

